apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -100,6 +101,16 @@ const providers: Provider[] = [
         }
       }
 
+      const limiter = rateLimit({
+        interval: 60 * 1000, // 1 minute
+        uniqueTokenPerInterval: 100, // Max 10 login retries per minute
+      });
+      const { isRateLimited } = await limiter.check(100, user.username); // 10 requests per minute
```

Comment:
- Can the make this function throw instead of returning a `isRateLimited`?
- Why it says `10 request per minute` and we're passing a `100` there?

Reply:
Sure, will change and update, I was testing more requests to see if that fixed the onboarding tests, but what should be the limit we allow? Should it be process.ENV based too in case we want to change this without updating the codebase? @zomars

Reply:
> Should it be process.ENV based too in case we want to change this without updating the codebase?

Not now